app/services/stripe_service/api/payments.rb

```diff
@@ -57,7 +58,7 @@ def create_preauth_payment(tx, gateway_fields)
       def cancel_preauth(tx, reason)
         payment = PaymentStore.get(tx[:community_id], tx[:id])
         seller_account = accounts_api.get(community_id: tx[:community_id], person_id: tx[:listing_author_id]).data
-        stripe_api.cancel_charge(tx[:community_id], payment[:stripe_charge_id], seller_account[:stripe_seller_id], reason)
+        stripe_api.cancel_charge(tx[:community_id], payment[:stripe_charge_id], seller_account[:stripe_seller_id], reason, {order_id: tx[:id]})
```

Comment:
Last argument is metadata, right? Same as above, let's prefix keys with `sharetribe_`.